doc/tutorials/plugins.md

```diff
@@ -11,45 +11,45 @@ and what various methods exists within one.
 
 ### The Interface ###
 
-All plug-ins use the same basic interface. This interface consists of five basic functions, 
```

Comment:
is there a possibility to remove a commit form the pull request? So I can make an extra pull request with this fixes?